docker-compose.yml

```diff
@@ -68,6 +68,7 @@ services:
       dockerfile: ./docker/images/e2e/Dockerfile
     image: marsha:e2e
     env_file:
+      - env.d/db
```

Comment:
We splitted the database environment variables to avoid database docker image recreation as we are changing things.
So, we need to add it to e2e to make it working locally.